imap_processing/swe/l1a/swe_l1a.py

```diff
@@ -12,7 +12,7 @@
 logger = logging.getLogger(__name__)
 
 
-def swe_l1a(file_path):
+def swe_l1a(file_path, data_version=None):
```

Comment:
should we make all data_version to be a required input? At this point, the cli input should give us correct version to use (given that we implement logic in the sds-data-manager to find the correct version through database).

Reply:
Ah ok. That makes total sense. I can update that in future PR. Thank you!